lib/url.js

```diff
@@ -69,11 +69,13 @@ const slashedProtocol = {
   'ftp': true,
   'gopher': true,
   'file': true,
+  'ni': true,
   'http:': true,
   'https:': true,
   'ftp:': true,
   'gopher:': true,
-  'file:': true
+  'file:': true,
+  'ni:': true
```

Comment:
from the original PR: all protocols are defined twice here, once with and once without a semicolon. I just replicated what was already there.